R/applyFilt.R

```diff
@@ -1118,8 +1118,23 @@ applyFilt.proteomicsFilt <- function(filter_object,
 
   # Check if min_num_peps is not NULL.
   if (!is.null(min_num_peps)) {
+    if (!is.null(pepe$e_data_remove)) {
```

Comment:
At this point, we've determined which peptides will be removed due to being associated with > 1 protein (redundant peps).  Now, when we check for proteins that dont have at least x peptides associated with them, we _do not_ take into account the redundant peps when seeing if we meet the threshold.